src/SmartComponents/PatchSet/PatchSet.js

```diff
@@ -114,9 +115,12 @@ const PatchSet = ({ history }) => {
         });;
     };
 
-    const CreatePatchSetButton = createPatchSetButton(setPatchSetState);
+    const { hasAccess } = usePermissionsWithContext([
+        'patch:*:*'
+    ]);
+    const CreatePatchSetButton = createPatchSetButton(setPatchSetState, hasAccess);
     const actionsConfig = patchSetRowActions(openPatchSetEditModal, handlePatchSetDelete);
-
+    const customActionsConfig = customActionsToggle;
```

Comment:
This seems a bit redundant, we could just use `customActionsToggle` below on line 158.